src/Sulu/Bundle/MediaBundle/Resources/config/services.xml

```diff
@@ -171,5 +171,23 @@
         <service id="sulu_media.security_context" class="Sulu\Component\Security\Authorization\SecurityCondition">
             <argument>sulu.media.collections</argument>
         </service>
+
+        <service id="sulu_media.system_collections.cache" class="Sulu\Component\Cache\ConfigCache">
```

Comment:
i will remove the own config cache class but the dependency injection i think is the better way. because then we can test the cache handling properly ... to load the file content i will use the `__toString` function.